<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>searchDialog</class>
 <widget class="QWidget" name="searchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>搜索</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_root">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_header">
     <item>
      <widget class="QLabel" name="lblTitle">
       <property name="text">
        <string>搜索</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignVCenter|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_header">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btnClose">
       <property name="text">
        <string>×</string>
       </property>
       <property name="toolTip">
        <string>关闭</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_search">
     <item>
      <widget class="QLineEdit" name="txtQuery">
       <property name="placeholderText">
        <string>输入关键字</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboScope">
       <item>
        <property name="text">
         <string>全部</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>联系人</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>群组</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>消息</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSearch">
       <property name="text">
        <string>搜索</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabResults">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <widget class="QWidget" name="tabUsers">
      <attribute name="title">
       <string>联系人</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_users">
       <item>
        <widget class="QListWidget" name="listUsers"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabGroups">
      <attribute name="title">
       <string>群组</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_groups">
       <item>
        <widget class="QListWidget" name="listGroups"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMessages">
      <attribute name="title">
       <string>消息</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_messages">
       <item>
        <widget class="QListWidget" name="listMessages"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_footer">
     <item>
      <widget class="QLabel" name="lblStatus">
       <property name="text">
        <string>请输入关键字进行搜索</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_footer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string>清空</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
